_posts/README.md

```diff
@@ -0,0 +1,89 @@
+# Creating post pages for the What's new feed
+
+The feed on our [What's new page](https://designsystem.digital.gov/about/whats-new/) is powered by the files in two different directories:
+- `/_posts`: These are the long format posts generated by [Jekyll posts](https://jekyllrb.com/docs/posts/). These files will automatically generate both a post preview for the news feed and a separate page for the post content.
+  - See the instructions in this document to learn how to [create a new post](#creating-a-new-post), [customize a post's metadata](#customizing-post-metadata), and follow [post conventions](#post-conventions).
+- `/_posts_short`: These are the short format previews that appear in the news feeds. These files do _not_ generate a related post page, and instead often link to different pages on our site or other locations.
+  - See the README in the /_posts_short directory for instructions on creating a short post
+
+## Creating a new post
+
+To create a new post, create a new branch then add a new file to this directory (`/_posts`) with a filename in the form:
+
+```
+YYYY-MM-DD-title.md
+```
+
+where:
+
+- `YYYY` is the 4-digit year (`2024`) of the publish date
+- `MM` is the 2-digit month, with a leading zero (`01` for January, `02` for February, etc.)
+- `DD` is the 2-digit day of the month, also with a leading zero
+- `title` is the so-called "slug" used in the post URL, and should be all lowercase with no spaces (e.g. `my-post-title`)
+
+>[!note]
+> The YYYY-MM-DD date set in the file name will:
+> - Display as the "date" in the post
+> - Sort the post in reverse chronological order (newest items first) in the news feed.
+
+See the [Jekyll documentation about creating posts](https://jekyllrb.com/docs/posts/#creating-post-files) for more info.
+
+Next, in the file's contents, copy and paste the following:
+
+```
+---
```

Comment:
Do we explain frontmatter for posts/short posts anywhere?

Are there frontmatter fields that we're missing?

Reply:
I added some content in [d3ff4cb](https://github.com/uswds/uswds-site/pull/3047/commits/d3ff4cbc31318a5d38b546a475a80169ce9f455f) that explains the sections of this template. Let  me know if you want additional changes.